app/uk/gov/hmrc/fileupload/read/envelope/model.scala

```diff
@@ -49,39 +50,16 @@ case class File(fileId: FileId,
                 metadata: Option[JsObject] = None,
                 rel: Option[String] = Some("file"))
 
-object Envelope {
-
-  type ContentTypes = String
-
-  val acceptedMaxItems: Int = 100
-  val acceptedMaxSize: String = "250MB" //250 * 1024 * 1024
-  val acceptedMaxSizePerItem: String = "100MB" //100 * 1024 * 1024
-  val acceptedContentTypes: List[ContentTypes] =
-                                List("application/pdf",
-                                     "image/jpeg",
-                                     "text/xml",
-                                     "text/csv",
-                                     "application/xml",
-                                     "application/vnd.ms-excel",
-                                     "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
-
-  val defaultMaxItems: Int = 100
-  val defaultMaxSize: String = "25MB" //25 * 1024 * 1024
-  val defaultMaxSizePerItem: String = "10MB" //10 * 1024 * 1024
-  val defaultContentTypes: List[ContentTypes] = List("application/pdf","image/jpeg","application/xml","text/xml")
-
-  val defaultConstraints =
-    EnvelopeConstraints(maxItems = defaultMaxItems,
-                        maxSize = defaultMaxSize,
-                        maxSizePerItem = defaultMaxSizePerItem,
-                        contentTypes = defaultContentTypes)
-
-  val acceptedConstraints =
-    EnvelopeConstraints(maxItems = acceptedMaxItems,
-                        maxSize = acceptedMaxSize,
-                        maxSizePerItem = acceptedMaxSizePerItem,
-                        contentTypes = acceptedContentTypes)
+case class EnvelopeConstraintsConfigure(acceptedMaxItems:Int,
```

Comment:
Confugure => Configuration

Reply:
acknowledged, will be changed in next commit